lib/src/presentation/scenes/habit/habit_row.dart

```diff
@@ -122,9 +125,9 @@ class _HabitRowState extends State<HabitRow> {
         width: _screenSize.width * 0.5,
         margin: EdgeInsets.only(left: _screenSize.width * 0.1),
         padding: EdgeInsets.only(
-            left: _screenSize.width > 1100
+            left: _screenSize.width > 1150
```

Comment:
@vpaniush would be great to extract these magic numbers into constants

Reply:
@bogdanustyak updated